### PR TITLE
normalize the order of import package

### PR DESCRIPTION
**Pull request by gy95**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
the import order of packages are confusing, i think we can put those import packages together.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by neolit123** [comment]
/lgtm
/approve
/ok-to-test

i can only approve the cmd/kubeadm change.

**Comment by gy95** [comment]
@derekwaynecarr PTAL, thanks

**Review by ehashman — COMMENTED** [review]
/lgtm
for pkg/probe I guess

@liggitt can you approve?

/priority backlog

File: pkg/probe/tcp/tcp.go [hunk 1]
```diff
@@ -21,9 +21,8 @@ import (
 	"strconv"
 	"time"
 
-	"k8s.io/kubernetes/pkg/probe"
-
 	"k8s.io/klog/v2"
```
Comment (liggitt) [inline pkg/probe/tcp/tcp.go:24]: the intent was to keep imports from this repo together... k8s.io/klog is an external repo

File: pkg/controller/namespace/namespace_controller.go [hunk 2]
```diff
@@ -34,10 +34,9 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/component-base/metrics/prometheus/ratelimiter"
+	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/namespace/deletion"
-
-	"k8s.io/klog/v2"
```
Comment (liggitt) [inline pkg/controller/namespace/namespace_controller.go:40]: the intent was to keep imports from this repo together... k8s.io/klog is an external repo

File: cmd/kubeadm/app/componentconfigs/kubelet.go [hunk 3]
```diff
@@ -23,13 +23,12 @@ import (
...
 	kubeletconfig "k8s.io/kubelet/config/v1beta1"
-	"k8s.io/kubernetes/cmd/kubeadm/app/util/initsystem"
-	utilpointer "k8s.io/utils/pointer"
-
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/features"
+	"k8s.io/kubernetes/cmd/kubeadm/app/util/initsystem"
+	utilpointer "k8s.io/utils/pointer"
```
Comment (liggitt) [inline cmd/kubeadm/app/componentconfigs/kubelet.go:31]: I think the intent was to keep imports from this repo together... k8s.io/utils is an external repo

File: pkg/controller/namespace/namespace_controller.go (hunk 2 above)
Comment (gy95) [reply pkg/controller/namespace/namespace_controller.go:40]: no, other imports also think it is k8s its own package, just as below
https://github.com/kubernetes/kubernetes/blob/3bd29bc53d5acb2ac794f4d150472f6a759f0e23/pkg/credentialprovider/aws/aws_credentials.go#L36-L40

File: pkg/probe/tcp/tcp.go (hunk 1 above)
Comment (gy95) [reply pkg/probe/tcp/tcp.go:24]: i think all packages of having prefix `k8s.io/` be better put together

**Comment by neolit123** [comment]
@gy95 

please remove the cmd/kubeadm changes from this PR.
i logged a separate issue for the kubeadm cleanup:
https://github.com/kubernetes/kubeadm/issues/2515

/remove-area kubeadm
/remove-sig cluster-lifecycle

**Comment by gy95** [comment]
> @gy95
> 
> please remove the cmd/kubeadm changes from this PR.
> i logged a separate issue for the kubeadm cleanup:
> [kubernetes/kubeadm#2515](https://github.com/kubernetes/kubeadm/issues/2515)
> 
> /remove-area kubeadm
> /remove-sig cluster-lifecycle

done

**Comment by ehashman** [comment]
Given @liggitt's comments I don't know that this PR really adds anything so I'm going to close it.

Please take a look at our [trivial edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits). We encourage you to take a look at confirmed issues and bugs or issues marked `help-wanted`.

/close

**Pull request closed without merging**